### ISSUE TITLE
Voxel visualization seems to fail with version 0.6.0

### ISSUE THREAD

**Issue by MiaoDX** [intel-isl/Open3D#941]
_____

**Describe the bug**
I failed to visualize the voxels with version 0.6.0 on my ubuntu PC, both py=3.5/3.7 (and at another machine). Turned to 0.5.0 works like a charm.

**To Reproduce**

```
from open3d import read_voxel_grid, draw_geometries
# https://github.com/intel-isl/Open3D/raw/master/examples/TestData/bathtub_0154_voxel.ply
voxel_read = read_voxel_grid("bathtub_0154_voxel.ply")
print(voxel_read)
draw_geometries([voxel_read])
```

**Expected behavior**

``` vi
Reading PLY: [========================================] 100%
VoxelGrid with 18733 voxels.
```
And there should be views, but not the case for 0.6.0

**Environment (please complete the following information):**
 - OS: [Ubuntu 16.04]
 - Python version: 3.7.1/3.5.6
 - Open3D version: 0.6.0
 - Is this remote workstation?: no
 - How did you install Open3D?:  both pip [AND] conda

**Additional context**
I am not so sure whether this is only me or not, but limited tests seem chances are that you may accidentally miss something when building the new version. Correct me if I am wrong.

And, thanks for the wonderful lib ^_^

**Comment by yxlao** [comment]
Thanks for reporting, this is related to updated OpenGL version -- OpenGL 3.1+ removed `GL_QUADS` support, thus our previous SimpleShaderForVoxelGrid is broken. 

This has been fixed in https://github.com/intel-isl/Open3D/pull/918, please compile from source to get this update now. It will be included in the 0.7 release.

**Comment by MiaoDX** [comment]
Yeah, thanks, I will close it now.
